connectors/src/connectors/google_drive/lib.ts

```diff
@@ -51,3 +53,34 @@ export async function registerWebhook(
     return new Err(new HTTPError(await res.text(), res.status));
   }
 }
+
+async function _getParents(
```

Comment:
Can we call it and the memoized version `getLocalParents` to make it clear that we're only going to the DB in contrast with the existing `getFileParentsMemoized` which goes to the network?

Reply:
sure